app.py

```diff
@@ -161,7 +161,11 @@ def process_error(error):
     if error.find("Your code has been rated at") > -1:
         return None
 
-    line_num = error.split(":")[1]
+    # Detect OS
+    if os.name == "nt":
+        line_num = error.split(":")[2]
+    else:
+        line_num = error.split(":")[1]
```

Comment:
Could you add a try and except block here just in case another operating system is not compatible?